spec/index.html

```diff
@@ -29,21 +29,19 @@
           { name: "Chimezie Ogbuji" },
         ],
 
-        github: { repoURL: "https://github.com/w3c/sparql-entailment", branch: "main"},
+        github: "https://github.com/w3c/sparql-entailment",
 
         shortName: "sparql12-entailment",
         xref: "web-platform",
         group: "rdf-star" ,
         wgPublicList: "public-rdf-star-wg",
 
-// Only for publication        
-//         prevVersion:          "http://www.w3.org/TR/2013/REC-sparql11-entailment-20130321",
-//         previousPublishDate:  "2013-03-21",
-//         previousMaturity:     "TR",
-//         prevRecURI:           "http://www.w3.org/TR/2013/REC-sparql11-entailment-20130321",
+      previousPublishDate:  "2013-03-21",
+      previousMaturity:     "REC",
+      prevRecShortname:     "sparql11-entailment",
         
-        errata:               "https://w3c.github.io/rdf-star-wg/errata",
-        edDraftURI:           "https://w3c.github.io/sparql12-entailment/",
+//        errata:               "https://w3c.github.io/sparql12-entailment/errata/",
```

Comment:
Personally, I think this need careful consideration.

There is a trade-off of splitting the errata per spec (editor centric) against one SPARQL errata (more reader centric?)

Reply:
It’s just that there’s a W3C tool that does this, but we could certainly modify something. Still, not really a concern until close to PR.

Reply:
PR or PR :-)
This is a PR.

Reply:
😄 Proposed Recommendation. https://www.w3.org/2021/Process-20211102/#transition-pr